packages/core/src/view/event/InternalMouseEvent.ts

```diff
@@ -191,7 +191,7 @@ class InternalMouseEvent {
   consume(preventDefault?: boolean) {
     preventDefault = preventDefault
       ? preventDefault
-      : this.evt instanceof TouchEvent || isMouseEvent(this.evt);
+      : (isMouseEvent(this.evt) || this.evt instanceof TouchEvent);
```

Comment:
I think it's more safe to add check for window.TouchEvent is not falsy, because in proposed approach if `isMouseEvent` will somehow return `false` Firefox/Safari will still fail.
 
```suggestion
      : (window.TouchEvent && this.evt instanceof TouchEvent) || isMouseEvent(this.evt);
```

Checked latest on Firefox/Chrome (desktop and touch emulations) and it works fine

Reply:
Yes you are right, this is also what is proposed in the resources pointed in #76